sources/libs/cc/ast/decl.h

```diff
@@ -9,6 +9,7 @@ typedef enum
 {
     CDECL_NIL,
 
+    CDECL_EMPTY,
```

Comment:
What does an empty declaration look like?

Reply:
for the moment it's just:
int i; <= variable declaration
; <= empty declaration